tests/integration/targets/nd_pcv/tasks/out-ospf-to-l2vrf.json

```diff
@@ -1 +1,359 @@
\ No newline at end of file
+{
```

Comment:
Any reason to make it pretty printed?

Reply:
Did that to be able to read what was happening in this file, then left it. Can change it back if you want me to?

Reply:
keep it as pretty print